### DISCUSSION TITLE
Difference about virtio-ramfb and virtio-gpu-gl-pci

### DISCUSSION THREAD

**Discussion by jdjingdian** [utmapp/UTM#3073] [UTM Help (macOS)] [answered]
Hi! I noticed that we can use  `GPU acceleration for OpenGL on Linux`. Can someone explain the difference about virtio-ramfb and virtio-gpu-gl-pci?  Which one can have a better performance? Thanks in advance.

**Comment by osy** [comment, marked as answer]
It’s the same but some distros don’t support virtio-gpu-gl-pci

**Reply by kode54** [reply]
Are both hardware accelerated by the host? I can't seem to get very good performance with virtio-ramfb.
